docs/config-file/v2.rst

```diff
@@ -283,7 +283,9 @@ sphinx.builder
 The builder type for the Sphinx documentation.
 
 :Type: ``string``
-:Options: ``html``, ``htmldir``, ``singlehtml``
+:Options: ``html``, ``dirhtml``, ``singlehtml``   
+..note::
+   The htmldir builder option was renamed to dirhtml to use the same name as sphinx. Configurations using the old name will continue working.
```

Comment:
This note should be below the line `Default html`

Reply:
Done